Dockerfile

```diff
@@ -1,73 +1,31 @@
-FROM debian:jessie
-MAINTAINER Reaction Commerce <admin@reactioncommerce.com>
+FROM node:8.9.4
+MAINTAINER Reaction Commerce <architecture@reactioncommerce.com>
 
-RUN groupadd -r node && useradd -m -g node node
+# Meteor
+ARG METEOR_VERSION
+ENV METEOR_VERSION ${METEOR_VERSION:-1.6.0.1}
 
 # Gosu
 ENV GOSU_VERSION 1.10
 
-# MongoDB
-ENV MONGO_VERSION 3.4.10
-ENV MONGO_MAJOR 3.4
-ENV MONGO_PACKAGE mongodb-org
-
-# PhantomJS
-ENV PHANTOM_VERSION 2.1.1
-
-# provide a way to detect if we're inside
+# provide a way for reaction-cli to detect if we're inside
 # an official Reaction Docker build
 ENV REACTION_DOCKER_BUILD true
 
 # build directories
 ENV APP_SOURCE_DIR /opt/reaction/src
 ENV APP_BUNDLE_DIR /opt/reaction/dist
-ENV BUILD_SCRIPTS_DIR /opt/build_scripts
+ENV BUILD_SCRIPTS_DIR /opt/reaction/scripts
```

Comment:
With multistage builds we can include the script actions directly in the Dockerfile. That would provide better build caching and comprehension and a simpler build setup.

Reply:
This entire image serves as a development/build environment for Reaction and is essentially cached at the point of release.  The only time this ever gets built is during the CI test/release process.  Nobody ever has to build this image repeatedly in development.